generators/server/templates/src/main/java/package/config/_MicroserviceSecurityConfiguration.java

```diff
@@ -101,6 +104,11 @@ public SecurityEvaluationContextExtension securityEvaluationContextExtension() {
     @Inject
     JHipsterProperties jHipsterProperties;
 
+    <%_ if (hibernateCache !== 'hazelcast') { _%>
```

Comment:
You're right, I changed the PR to always load the discovery client because it isn't bound to Hazelcast.  

When does this cause other issues?  I didn't have any issues in dev/prod/test profiles.